jest.config.js

```diff
@@ -1,21 +1,57 @@
-module.exports = {
-  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+const commonOptions = {
   transform: {
     "^.+\\.tsx?$": "ts-jest",
   },
-  testRegex: "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
   testPathIgnorePatterns: [
     "\\.snap$",
     "/node_modules/",
     "/dist/",
     "/examples/e2e/",
   ],
-  testEnvironment: "jsdom",
+  setupFiles: ["<rootDir>/jest-setup.ts"],
+};
+
+const WEB_PACKAGES = [
+  "flipper-plugin-android-performance-profiler",
+  "web-reporter-ui",
+];
+const NODE_PACKAGES = [
+  "android-performance-profiler",
+  "appium-helper",
+  "appium-test-cases",
+  "aws-device-farm",
+  "e2e-performance",
+  "eslint-plugin-flashlight-eslint-rules",
+  "measure",
+  "reporter",
+  "shell",
+  "web-reporter",
+];
```

Comment:
How do we remember to add new packages here? Do we have a standard of package creation?

Reply:
when you add a test and `jest` doesn't run on the test, I'd check the `jest.config` I guess 😅 
i don't think we'll add so many packages, but it could be a good idea in the future indeed!